Header header
float32 minimum_value
float32 maximum_value
